OpenOversight/app/utils.py

```diff
@@ -473,6 +474,32 @@ def create_description(self, form):
         date_updated=datetime.datetime.now())
 
 
+def conditionally_rotate_image(image):
+    try:
+        # NOTE: Pillow >= 6.0 provides `ImageOps.exif_transpose` which acts like the below code
+
+        # find the orientation key
+        for orientation in ExifTags.TAGS.keys():
+            if ExifTags.TAGS[orientation] == 'Orientation':
+                break
+
+        exif = dict(image._getexif().items())
+
+        if exif[orientation] == 3:
+            current_app.logger.info('image had EXIF orientation 3; rotating 180 degrees')
```

Comment:
These loggers might be useless. They don’t really provide any actionable information and they expose a behavior the user would otherwise assume “just works.”

Reply:
I don't see the detriment to keeping them. It would be interesting for admins to see how often we run across these EXIF rotation issues.